### PR TITLE
[3.0][docs] add tutorial for mysql to doris pipeline.

### PR DESCRIPTION
**Pull request by lvyanquan**
CC.
@Jiabao-Sun @JNSimba @PatrickRen @loserwang1024 .

### CODE REVIEW

File: docs/content/快速上手/mysql-doris-pipeline-tutorial-zh.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+   username: root
+   password: 123456
+   tables: app_db.\.*
+   server-id: 5400-5404
+   server-time-zone: UTC
+   
+   sink:
+   type: doris
+   fenodes: 127.0.0.1:8030
+   benodes: 127.0.0.1:8040
```
Comment (JNSimba) [inline docs/content/快速上手/mysql-doris-pipeline-tutorial-zh.md:187]: This benodes is not required, can it be removed? Simplify the configuration

File: docs/content/快速上手/mysql-doris-pipeline-tutorial-zh.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+   ################################################################################
+   source:
+   type: mysql
+   hostname: localhost
+   port: 3306
+   username: root
+   password: 123456
+   tables: app_db.\.*
+   server-id: 5400-5404
+   server-time-zone: UTC
```
[the remark this answers is not in the document]
Comment (lvyanquan) [reply docs/content/快速上手/mysql-doris-pipeline-tutorial-zh.md]: Thanks a lot for pointing out this, almost caused a misleading.

File: docs/content/快速上手/mysql-doris-pipeline-tutorial-zh.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+   table.create.properties.light_schema_change: true
+   table.create.properties.replication_num: 1
+
+   route:
+   - source-table: app_db.orders
+     sink-table: ods_db.ods_orders
+   - source-table: app_db.shipments
+   sink-table: ods_db.ods_shipments
+   - source-table: app_db.products
+   sink-table: ods_db.ods_products
```
[the remark this answers is not in the document]
Comment (lvyanquan) [reply docs/content/快速上手/mysql-doris-pipeline-tutorial-zh.md]: address it.

**Review by JNSimba — APPROVED** [review]
LGTM

**Review by Jiabao-Sun — APPROVED** [review]
+1

**Comment by Jiabao-Sun** [comment]
Hi @lvyanquan, could you help make a BP-3.0?

**Comment by lvyanquan** [comment]
done in https://github.com/ververica/flink-cdc-connectors/pull/2843.

**Pull request merged**